db/migrate/20220811093403_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :post, foreign_key: true, null: false
+      t.references :user, foreign_key: true, null: false
+      t.text :Text
```

Comment:
- [ ] It is a good practice to be consistent in naming the variables, and it is recommended to write them with small letter, because capital letters in Ruby are constants, you or others may get very confused in further development

please change `t.text :Text` to `t.text :text`

Reply:
Implemented 👍